libproxy/cmake/libproxy.cmk

```diff
@@ -1,4 +1,9 @@
-add_library(libproxy SHARED extension_config.cpp extension_pacrunner.cpp extension_wpad.cpp proxy.cpp url.cpp ${LIBPROXY_SOURCES})
+if(BUILD_SHARED_LIBS)
```

Comment:
Can you do

```
set(LIBPROXY_SOURCES ${LIBPROXY_SOURCES} extension_config.cpp extension_pacrunner.cpp extension_wpad.cpp proxy.cpp url.cpp)
```

instead of this duplication?

Reply:
done https://github.com/libproxy/libproxy/pull/195/commits/c753fcfe5285e98d9496f64a3b41dc48b986c1f5